build.gradle.kts

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 group = "uk.nhs.tis.trainee"
-version = "1.11.3"
+version = "1.12.1"
```

Comment:
Odd to jump a major version now, but it should have been set to 1.12.0 for the 12W placement PR - somehow it got lost there 🤦